packages/fast-usdc/src/cli/transfer.js

```diff
@@ -39,13 +40,16 @@ const transfer = async (
       { chainName: 'agoric', rpcAddrs: [pickEndpoint(netConfig)] },
     );
     const agoricAddr = await queryFastUSDCLocalChainAccount(vstorage, out);
-    const appendedAddr = `${agoricAddr}?EUD=${destination}`;
-    out.log(`forwarding destination ${appendedAddr}`);
+    const encodedAddr = encodeAddressHook(agoricAddr, {
+      EUD: destination,
+    });
```

Comment:
minor style point: I'd much rather see:

```suggestion
    const encodedAddr = encodeAddressHook(agoricAddr, { EUD });
```

with the arg renamed to `EUD`. Do we need 2 names for it?